libs/wingsdk/src/core/app.ts

```diff
@@ -171,182 +163,6 @@ export abstract class App extends Construct {
   }
 }
 
-/**
- * An app that knows how to synthesize constructs into Terraform configuration
- * using cdktf. No polycon factory or Terraform providers are included.
- */
-export abstract class CdktfApp extends App {
```

Comment:
I've moved this outside of `core.App` so that we won't have CDKTF APIs leaking out of the core module. It's not directly related but technically if we want to do the same trick for `cdktf` in the future, this will be required.

Also, it does't make sense at all that this class is here :-)